### ISSUE TITLE
No response after successful installation

### ISSUE THREAD

**Issue by fzc1985** [zitmen/thunderstorm#69]
I have used ThunderSTORM before on my old computer. And successfully installed it on my new computer (the plugin is shown, and every tabs are shown). However, there is no response after I hit any of the tabs in the plugin. 
Is this a compatible problem with Win 10?

**Comment by zitmen** [comment]
Yes, it is compatible on Windows 10.
What version of ImageJ and JRE do you use? Do other plugins work well?
If you open Window -> Console and then try to start the ThunderSTORM analysis, do you see any message in the Console or Log tabs?

**Comment by fzc1985** [comment]
Dear Martin,
Thank you for your reply!
I have figured out this problem right now, but don't understand the
mechanism behind it. I want to share my experience and let you know maybe
there are some bugs.
When I see this issue, I do:
1. Go to the main page:  https://github.com/zitmen/thunderstorm
2. Select "Clone or download". I tried to choose different versions
(branches/tags), including master, pr/54, v1.3.
3. I rename the downloaded zip file to Thunder_STORM.jar as said in the
guide.
4. Copy it to the plugin folder in FIJI.
5. The ThunderSTORM plugin is shown, but no response after I click any
tabs. Other plugins work fine.

Then one of my friends send me the link:
https://github.com/zitmen/thunderstorm/releases/tag/v1.3
After I directly download the jar file and copy it to the plugin folder, it
works.

best,

Zhichao
